.github/workflows/deployment-test.yaml

```
@@ -53,24 +50,34 @@ on:
+# wait behind cloud provisioning.
+concurrency:
+  group: >-
+    ${{ (github.event_name == 'schedule'
+          || github.event.inputs.mode == 'full-deployment'
+          || (github.event_name == 'pull_request'
+              && github.event.action == 'labeled'
+              && github.event.label.name == 'ci:azure-deployment'))
+        && 'deployment-test-azure'
+        || format('deployment-test-azure-{0}', github.run_id) }}
+  cancel-in-progress: false
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)